Source/WebCore/page/Quirks.cpp

```diff
@@ -2456,6 +2465,19 @@ static void handleSFUSDQuirks(QuirksData& quirksData, const URL& quirksURL, cons
     quirksData.shouldBypassAsyncScriptDeferring = true;
 }
 
+static void handleSlackQuirks(QuirksData& quirksData, const URL& quirksURL, const String& quirksDomainString, const URL& documentURL)
+{
+    if (quirksDomainString != "slack.com"_s)
+        return;
+
+    UNUSED_PARAM(quirksURL);
+    UNUSED_PARAM(documentURL);
```

Comment:
You can just omit parameter names entirely, since they are not used in any code path.